CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ You can find its changes [documented below](#070---2021-01-01).
   context-methods to implement disabled ([#1632] by [@xarvic])
 - `LifeCycle::BuildFocusChain` to update the focus-chain ([#1632] by [@xarvic])
 - `DisabledIf` widget wrapper to disable based on the state of Data and Env ([#1702] by [@xarvic])
+- `chrono` feature with `Data` support for [chrono](https://docs.rs/chrono/) types ([#1743] by [@r-ml])
```

Comment:
you'll also need to add the links to `[#1743]` and `[@r-ml]`, below.

Reply:
I think I did it correctly

Reply:
yep looks good!